behaviors/README.md

```diff
@@ -9,20 +9,137 @@ behaviors/
 
 ## Defined in the schema
 
-Behaviors are added to fields in your `schema.yaml`. They are an array of objects, with a `fn` property and any number of arguments:
+Behaviors are added to fields in your `schema.yaml`. They are an array of strings and/or objects (with a `fn` property and any number of arguments):
 
 ```yaml
-my-field:
+myField:
   _has:
-    -
-      fn: text
-      required: true
-      placeholder: This is my placeholder
+    - text
     -
       fn: soft-maxlength
       maxLength: 80
 ```
 
+### Field Properties
+
+Besides `_has` (which is an array of behaviors), fields can have certain properties. These are prefixed with underscores.
```

Comment:
"Besides" kind of made me think for a minute. "In addition to" sounds clearer to me.